docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -222,6 +220,12 @@ updates to its `.spec.template` will not be propagated to its Pods.
 In most cases you will not need to use a partition, but they are useful if you want to stage an
 update, roll out a canary, or perform a phased roll out.
 
+## Selectors
+
+Same as other controllers, the StatefulSet utilizes selectors to identify pods that it should manage. For a pod to be managed by a StatefulSet, the pod needs to have labels matching the StatefulSet's selectors. However, the StatefulSet does not keep track whether a pod is created by it or another controller. The pods will persist when the StatefulSet is deleted (except when the user performs a [cascading deletion](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/)).
```

Comment:
@kow3ns IMO, it may be better to discuss what a selector is before moving on to indicate that there is no more selector defaulting in next paragraph.

Reply:
Label selectors are explained elsewhere in the documentation. We don't need to be repetitive. You can link to the Label selector documentation without reiterating the information there. 

This paragraph is also not correct. StatefulSet uses a combination of label selectors, controller references, and pod names to determine which Pods it should manage.

Reply:
Sure, i will put a link there instead of explaining about the label selectors.